src/pages/Home.jsx

```diff
@@ -18,74 +55,78 @@ function Home() {
               <ul className="nav nav-pills outline-active">
                 <li className="nav-item">
                   <button
+                    onClick={() => setActiveTag(null)}
                     type="button"
-                    className="nav-link" // Add active if not feed and not tag
+                    className={classNames('nav-link', { active: !activeTag })}
                   >
                     Global Feed
                   </button>
                 </li>
-                <li className="nav-item">
-                  <a className="nav-link active"># {faker.lorem.word()}</a>
-                </li>
+                {activeTag && (
+                  <li className="nav-item">
+                    <a className="nav-link active"># {activeTag}</a>
+                  </li>
+                )}
               </ul>
             </div>
-            <div className="article-preview">
-              <div className="article-meta">
-                <a>
-                  <img src={faker.image.avatar()} />
-                </a>
-                <div className="info">
-                  <a className="author">{faker.internet.userName()}</a>
-                  <span className="date">{new Date(faker.date.past()).toDateString()}</span>
+
+            {articles?.map((article) => (
+              <div className="article-preview" key={article?.slug}>
+                <div className="article-meta">
+                  <a>
+                    <img src={article?.author?.image} />
+                  </a>
+                  <div className="info">
+                    <a className="author">{article?.author?.username}</a>
+                    <span className="date">{new Date(article?.createdAt).toDateString()}</span>
+                  </div>
+                  <button
+                    type="button"
+                    className={classNames('btn btn-sm', {
+                      'btn-outline-primary': !article?.favorited,
+                      'btn-primary': article?.favorited,
+                    })}
+                    disabled={false}
+                  >
+                    <i className="ion-heart" />
+                    &nbsp; {article?.favoritesCount}
+                  </button>
                 </div>
-                <button
-                  type="button"
-                  className="btn btn-sm btn-outline-primary" // Change to btn-primary if favorited
-                  disabled={false}
-                >
-                  <i className="ion-heart" />
-                  &nbsp; {faker.datatype.number()}
-                </button>
+                <a className="preview-link">
+                  <h1>{article?.title}</h1>
+                  <p>{article?.description}</p>
+                  <span>Read more...</span>
+                  <ul className="tag-list">
+                    {article?.tagList.map((tag) => {
+                      ;<li className="tag-default tag-pill tag-outline">{tag}</li>
```

Comment:
You're not returning anything from the `map` function.
```suggestion
                      return <li className="tag-default tag-pill tag-outline">{tag}</li>
```